geometry_msgs/Point32[] towers